notebooks/HASP/WavelengthAdjustment/WavelengthAdjustment.ipynb

```diff
@@ -0,0 +1,2632 @@
+{
```

Comment:
<p>I see you are passing Path objects for the data_path. You can see those from the function invocation with stis_data and cos_data, both of which are path objects. I would suggest you change the docstring to reflect that:</p>

<pre class="ql-syntax" spellcheck="false">Input:
Path data_path: Path object to the directory where the files should be moved.
</pre><p>You can also take this if block out of the try,except block:</p><p>mast_path = data_path / "mastDownload/HST/"</p>

<pre class="ql-syntax" spellcheck="false">&nbsp;&nbsp;&nbsp;&nbsp;# Check if mastDownload exists
&nbsp;&nbsp;&nbsp;&nbsp;if not os.path.exists(mast_path):
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;print(f"Directory {mast_path} doesn't exist.")
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;return
 try:.....
</pre><p> </p><p>You can optimize the loops:</p>

<pre class="ql-syntax" spellcheck="false">try:
  for obs_id_path in mast_path.iterdir():
  	for file in obs_id_path.glob(obs_id_path + "*fits"):
  		shutil.move(str(file), data_path / filename)
  if mast_path.exists():
  	shutil.rmtree(mast_path)
except:....
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>